src/components/CreatePost/CreatePost.css

```diff
@@ -0,0 +1,137 @@
+.create-new-post
+{
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+}
+
+.whats-happening
+{
+  font-family: Comfortaa;
+  font-size: 16px;
+  padding: 1rem;
+  resize: none;
+  height: 10rem;
+  border: none;
+  outline: none;
+}
+
+.whats-happening:focus
+{
+  border: none;
+}
+
+.create-post-options-container
+{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 1rem;
+  padding: 1rem;
+  border-top: 1px solid var(--light-grey-2);
+}
+
+.create-post-options
+{
+  display: flex;
+  gap: 1rem;
+}
+
+.post-options-right-container
+{
+  display: flex;
+  align-items: center;
+}
+
+.create-post-icons-container
+{
+  padding: 6px;
+  border-radius: 50%;
+}
+
+.create-post-icons-container:hover
+{
+  background-color: var(--grey);
+}
+
+.create-new-post-icons
+{
+  font-size: 25px;
```

Comment:
Suggestion: if you can, try to convert this into rems, for better responsive font..

Reply:
Done